app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -327,7 +319,7 @@ def center_toolbar_filename_ops
   def center_toolbar_filename_report
     if x_active_tree == :db_tree
       node = x_node
-      if node == "root" || node == "xx-g"
+      if %w(node xx-g).include?(node)
```

Comment:
Typo, `node` should be `root` - causes https://github.com/ManageIQ/manageiq-ui-classic/issues/5153